### PR TITLE
feat: add ftc completion

### PR DESCRIPTION
**Pull request by el2zay**

### CODE REVIEW

**Comment by el2zay** [comment]
I have read the CLA Document and I hereby sign the CLA

File: README.md [hunk 1]
```diff
@@ -1,205 +1 @@
-<p align="center">
```
Comment (AlanSyue) [inline README.md:1]: Why clear the content of the README file 🤔

Comment (el2zay) [reply README.md:1]: I didn't touch it

File: .vscode/settings.json [hunk 2]
```diff
@@ -3,5 +3,8 @@
   "editor.codeActionsOnSave": {
     "source.fixAll.eslint": true
   },
-  "editor.formatOnSave": true
+  "editor.formatOnSave": true,
+  "[typescript]": {
```
Comment (AlanSyue) [inline .vscode/settings.json:7]: The project currently has a unified formatter. May I ask what is the purpose of adding this line? Thanks!

Comment (el2zay) [reply .vscode/settings.json:7]: I think it was inserted automatically by a plugin, I didn't touch anything except my own file.  I delete the modification?

Comment (AlanSyue) [reply .vscode/settings.json:7]: OK, I think there's no need to make this change. Thank you.

**Comment by AlanSyue** [comment]
I couldn't find any documentation related to the "ftc" command. Would it be possible for you to provide it if it's convenient?

**Comment by fedeci** [comment]
Also, docs should be standardized in US english language, can you translate them please?

**Comment by el2zay** [comment]
Actually I have not made any documentation on freetranscli because it is not yet completely finished. In the autocompletion everything is in French because the service can only be used by French people for the moment (because the API used is the one of a French operator). 

If you want a documentation of ftc and it's necessary tell me.

File: src/ftc.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
+const completionSpec: Fig.Spec = {
+  name: "ftc",
+  description: "",
```
Comment (fedeci) [inline src/ftc.ts:3]: Here a description should be provided:
```suggestion
  description: "",
```

File: src/ftc.ts [hunk 4]
```diff
@@ -0,0 +1,50 @@
+const completionSpec: Fig.Spec = {
```
Comment (fedeci) [inline src/ftc.ts:1]: @el2zay I would like to get a reference url to the cli tool download page/github repo/website as a comment on the top of this file.

File: .vscode/settings.json (hunk 2 above)
Comment (fedeci) [reply .vscode/settings.json:7]: Friendly ping on this :)

**Review by fedeci — CHANGES_REQUESTED** [review]
Here are some requested changes to address before merging this, I am still not 100% sold on the French descriptions.

**Review by el2zay — COMMENTED** [review]
I think everything is good

**Comment by el2zay** [comment]
Why ?

**Pull request closed without merging**